app/controllers/api/v1/bookmarks/check_controller.rb

```diff
@@ -1,6 +1,7 @@
 class Api::V1::Bookmarks::CheckController < Api::V1Controller
   # GET /api/v1/bookmarks/check
   def index
+    return head :bad_request unless params[:url]
```

Comment:
out of 200 line changes, is this the single line that fixes the bug?

Reply:
Nah that'd be lines [43](https://github.com/transientBug/transientbug-rails/pull/36/files#diff-bb9196b2f0cd7ff11bce257acea5eb92R43) and [51](https://github.com/transientBug/transientbug-rails/pull/36/files#diff-bb9196b2f0cd7ff11bce257acea5eb92R51) that fixes the auth bug, although this was causing a weird behavior if you managed to make it past the auth without an URL parameter. The rest of the changes is unfortunately just the doc rerendering, which is something I still need to break out into a docker build phase.